DEVELOPMENT.md

```diff
@@ -220,17 +220,17 @@ Environment variables
 TEST_WORK is used by registry server unit-test code to specify path to certificates in test system. 
 Note, THIS IS NOT USED IN PRODUCTION
 
-Notes about switching DeviceMode
+Notes about switching device mode
 ================================
 
-If DeviceMode is switched between LVM and Direct(ndctl), please keep
+If device mode is switched between LVM and direct(ndctl), please keep
 in mind that PMEM-CSI driver does not clean up or reclaim Namespaces,
-therefore Namespaces plus other related context (possibly LVM state)
+therefore Namespaces plus other related context (LVM state)
```

Comment:
As we are now starting to work on capitalization, let me also bring up "Namespace" - why with a capital letter? https://github.com/pmem/ndctl/blob/master/Documentation/ndctl/ndctl-create-namespace.txt uses it as a normal English word, i.e. lowercase.

We can also defer this - your choice.

Reply:
good point, let's change Namespace and Region. I recall I was thinking to stress that these are objects that are created. But I agree, this probably adds more confusion than it helps. I will change.